lib/index.js

```diff
@@ -33,10 +34,15 @@ co(function * () {
 		fly.emit('fly_run', {path: fly.file}).start(tasks)
 	}
 }).catch(function (e) {
-	if (e.code === 'ENOENT') {
-		utils.error('No Flyfile? See the Quickstart guide → git.io/fly-quick')
-	} else if (e.code === 'UNKNOWN_OPTION') {
-		utils.error('Unknown Flag: -' + e.key + '. Run fly -h to see available options.')
+	if (e.type === errorTypes.UnknownOption) {
+		utils.log('Unknown option ' + e.key + '. Run fly -h to see available options')
+	} else if (e.type === errorTypes.InvalidKey) {
+		utils.log([
+			'Invalid options\n',
+			'Note, options should start with -',
+			'and can not contain any special symbols, like {}*#?',
+			''
+		].join('\n'))
```

Comment:
Let's yell "Invalid Options!" with `utils.error` & then add the helpful bit "Options should start with..." with `utils.log`.